tools/BugReportTool/BugReportTool/RegistryUtils.h

```diff
@@ -8,3 +8,4 @@
 #include <Windows.h>
 
 void reportRegistry(const std::filesystem::path& tmpDir);
+void reportAdminFlags(const std::filesystem::path& tmpDir);
```

Comment:
Let's take the time to fix all functions names, they must be CamelCase with the first letter uppercase.